requirements/py3_dev.txt

```diff
@@ -1,3 +1,4 @@
 -r py3_common.txt
 -r dev.txt
 -r mypy.txt
+-r twisted.txt
```

Comment:
Note that after changing the requirements.txt files, you have to rerun `tools/provision.py` to rebuild the virtualenv to reflect the changes.